app/backend/wells/views_v2.py

```diff
@@ -97,7 +97,7 @@ def get_queryset(self):
                 raise NotFound(f'Unknown aquifer {intersects_aquifer_id}')
 
             # Simplify polygon and expand it by 1km
```

Comment:
Does this comment still reflect the code?

Reply:
It does. I changed the value of the simplification to match the amount of simplification that is done when an aquifer geom is saved.